cell/cell.go

```diff
@@ -5,10 +5,13 @@ package cell
 
 import (
 	"log/slog"
+	"time"
 
 	"go.uber.org/dig"
 )
 
+const logThreshold = 100 * time.Millisecond
```

Comment:
The main debate for me is this. I figured that it may be useful to have a threshold so that longer-running tasks would actually show up in the logs... but I'm not sold on this approach. It's also awkward to declare a private threshold here inside the library. Probably better than spamming the logs for now. :shrug:

Reply:
How about we make this configurable in `hive.Options` and have it disabled by default? I think it's better that we log a bunch by default as it's useful when learning to use the library.

Passing this to invoke is pretty easy, just add it as an argument to the `invoke` function that's added to `InvokerList`. For `Lifecycle` perhaps we could just have it as an exported field in `DefaultLifecycle` and we can set it in `hive.NewWithOptions`.